lib/heroku/kensa/check.rb

```diff
@@ -290,6 +298,73 @@ def call!
           reader, writer = IO.pipe
         end
 
+        if data[:async]
+          child = fork do
+            reader.close
+            server = TCPServer.open(7779)
+            client = server.accept
+            writer.write(client.readpartial(READLEN))
+            client.write("HTTP/1.1 200 OK\r\nContent-Length: 0\r\n\r\n")
+            client.close
+            writer.close
+          end
+          sleep(1)
+        end
```

Comment:
uhhhh, not really, haha. I brought it over from `ProvisionCheck`. I'm actually not sure if it's even needed or does anything in `ProvisionCheck`. I'm going to remove it from here.